src/record.rs

```diff
@@ -30,6 +37,7 @@ enum RecordType {
     Truncate = 1,
     Touch = 2,
     DeleteQueue = 3,
+    AppendRecords = 4,
```

Comment:
Should we ditch `AppendRecord`?

Reply:
Quickwit does not seem to need it, and a `AppendRecords` with a single record is something like 10 bytes wasted. I think we could remove it (but probably keep functions to create a single record, now generating an `AppendRecords`)

Reply:
that sounds like a plan